templates/aws/vm.cfn.template.yml

```diff
@@ -39,7 +36,5 @@ Resources:
         - !Ref SECURITYGROUPID 
       PropagateTagsToVolumeOnCreation: true
       Tags: 
-        - Key: "LansaVersion"
```

Comment:
As per your earlier feedback "If you just tag the stack with LansaVersion, all the resources will automatically be tagged. Remove all the individual resource tags" we will be tagging the stack and it will pass the tags to resources as well.

Reply:
Agreed